spec/fixtures/solr_documents/esri-image-map-layer.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
It seems that some of these are two spaced (`esri-dynamic-map-single.json`) and others 4 spaced (`esri-image-map-layer.json`). Can we standardize on 2 spaces?